php/WP_CLI/Runner.php

```diff
@@ -1392,6 +1409,12 @@ private function auto_check_update() {
 		// In case the operation fails, ensure the timestamp has been updated.
 		$cache->write( $cache_key, time() );
 
+		// Ask before doing remote request.
```

Comment:
Can you remove this, please, and open an issue to discuss changes to the update process before making them?